tests/field/test_selffieldforces.py

```diff
@@ -205,37 +205,59 @@ def test_update_points(self):
 
         # This test is incomplete.
 
-    def test_forces_taylor_test(self):
+    def test_meansquaredforces_taylor_test(self):
         """Verify that dJ matches finite differences of J"""
         # The Fourier spectrum of the NCSX coils is truncated - we don't need the
         # actual coil shapes from the experiment, just a few nonzero dofs.
-        curves, currents, axis = get_ncsx_data(Nt_coils=2)
 
-        # The next line is needed temporarily because ForceOpt doesn't yet
-        # account for derivatives with respect to currents. This next line
-        # should be removed once that derivative is added.
+        curves, currents, axis = get_ncsx_data(Nt_coils=2)
+        coils = [Coil(curve, current) for curve, current in zip(curves, currents)]
         [current.fix_all() for current in currents]
```

Comment:
It would be nice to get the Taylor test working with the currents unfixed. Does this work?

Reply:
it does!